04-issues-analyze.R

```diff
@@ -52,8 +52,13 @@ issues_vs_first_member_comment |>
   identity()
 
 # Slow issues
+bizdays::load_rmetrics_calendars(year = 2013:2024)
+
+try <- issues_vs_first_member_comment |>
+  mutate(time = bizdays::bizdays(from = issue_created_at, to = created_at, cal = "Rmetrics/ZURICH"))
+
 issues_vs_first_member_comment |>
-  mutate(time = as.numeric(created_at - issue_created_at) / 86400) |>
+  mutate(time = bizdays::bizdays(from = issue_created_at, to = created_at, cal = "Rmetrics/ZURICH")) |>
```

Comment:
how lucky are we that "zurich" is a built-in calendar. :smile_cat: 

of course you might have responded to issues on non business days but I still think it's a nicer measure, not only because it decreases the amount of days.